### ISSUE TITLE
Version 1.0.0 jar not working on jdk 1.7

### ISSUE THREAD

**Issue by pratts** [seancfoley/IPAddress#14]
Hello,
We are using java 7 on our systems and need v1.0.0 library for our IPv6 related requirements.
While trying to use the library I got the following error:
`Exception in thread "main" java.lang.UnsupportedClassVersionError: inet/ipaddr/IPAddressString : Unsupported major.minor version 52.0`

While trying to build the same using ant using task "create class jar", no files in ipv4, ipv6 and subdirectories in format folder are compiled and packaged. I get the following error during compilation:
`compile:
    [javac] /home/IPAddress/IPAddress/build.xml:47: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds

    [javac] Compiling 1 source file to /home/IPAddress/IPAddress/bin

    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.7

    [javac] /home/IPAddress/IPAddress/src/inet/ipaddr/IPAddressSection.java:1051: error: local variable skipThis is accessed from within inner class; needs to be declared final

    [javac] 				boolean done = skipThis;

    [javac] 				               ^

    [javac] /home/IPAddress/IPAddress/src/inet/ipaddr/IPAddressSection.java:1085: error: local variable skipThis is accessed from within inner class; needs to be declared final

    [javac] 				if(skipThis) {

    [javac] 				   ^

    [javac] 2 errors

    [javac] 1 warning

    [javac] Compile failed; see the compiler error output for details.
`

Kindly help.
Thank you!

**Comment by seancfoley** [comment]
Yes, class file version 52.0 corresponds to Java 8, so it looks like the jar you have been using was compiled as Java 8 binaries, it is quite possible that is what I checked into github even though the source is java 7 compatible.  
Yes, it looks like you have discovered a minor issue when compiling as Java 7, I just tried it myself.  It is a trivial fix, not sure how that small issue crept in there, it could simply be that the latest compiler is a bit different than the one I used previously.  There appear to be no other issues when compiling for Java 7, at least with the compiler I am using now.  I've checked in a fix:
https://github.com/seancfoley/IPAddress/commit/e3b0e8f269dc580ccd3049013c6d16d2f93c7a25
I can do a new release on the v1 branch with this change and check into github a java 7 jar for that release.
